content/influxdb/v1.8/query_language/explore-data.md

```diff
@@ -3140,6 +3140,16 @@ Use a subquery to apply a query as a condition in the enclosing query.
 Subqueries offer functionality similar to nested functions and SQL
 [`HAVING` clauses](https://en.wikipedia.org/wiki/Having_%28SQL%29).
 
+{{% warn %}}
+Subqueries **do not support** WHERE or OFFSET. Include **WHERE or OFFSET outside a subquery**. For example:
```

Comment:
This isn't true. Subqueries do support `WHERE` and `OFFSET`, but we recommend **time constraints** be on the outermost.

It might be worth it to just fix this in the query engine as it doesn't make logical sense. But sometimes where clauses need to be in subqueries in particular when filtering based on an intermediary result.